src/test/java/com/example/moviemanager/controller/MovieControllerTest.java

```diff
@@ -28,6 +31,9 @@
 @WebMvcTest
 public class MovieControllerTest {
```

Comment:
Since validations were moved to controller layer I would add parameterized test which tests each of them